cdap-e2e-tests/src/e2e-test/resources/test.csv

```diff
@@ -0,0 +1,3 @@
+Name,Id,Age
```

Comment:
Add this file in testdata folder, and provide that file path to be picked in scenario